src/krustlet-wascc.rs

```diff
@@ -25,7 +25,7 @@ async fn main() -> anyhow::Result<()> {
     module_store_path.push("modules");
     let store = FileModuleStore::new(client, &module_store_path);
 
-    let provider = WasccProvider::new(store, &config).await?;
```

Comment:
@thomastaylor312 should the Krustlet config object just contain a kube config?

Reply:
So the one problem with moving the config loading logic to the config (unless we make it a lazy load) is that you end up requiring a kubeconfig to even show the `--help` text. Because that code would be run while parsing the flags and returning the config